docs/dev/new_check_howto.md

```diff
@@ -318,6 +368,20 @@ Our check sends a Service Check though, so we need to add it to the `service_che
 ]
 ```
 
+Find below the description for each attributes -each one of them is mandatory- of your `service_checks.json` file:
+
+| Attribute       | Description                                                                                |
+| ----            | ----                                                                                       |
+| `agent_version` | Minimum Agent version supported.                                                           |
+| `integration`   | Integration name.                                                                          |
+| `check`         | Name of the Service Check                                                                  |
+| `statuses`      | Different status of the service check. To choose between: `ok`, `warning`, and `critical`. |
```

Comment:
Better IMO: `List of different status of the check, to choose among`. `unknown` is also a possibility